viewer/Viewer.cpp

```diff
@@ -150,6 +180,53 @@ namespace Enki
 		}
 	}
 	
+	void ViewerWidget::addManagedObjectsAlias(const std::type_info* key, const std::type_info* value)
+	{
+		managedObjectsAliases[key] = value;
+	}
+
+	World* ViewerWidget::getWorld()
+	{
+		return world;
+	}
+
+	QVector3D ViewerWidget::getPointedPoint()
+	{
+		return pointedPoint;
+	}
+
+	PhysicalObject* ViewerWidget::getPointedObject()
+	{
+		return pointedObject;
+	}
+
+	PhysicalObject* ViewerWidget::getSelectedObject()
+	{
+		return selectedObject;
+	}
+
+	bool ViewerWidget::isTrackballActivated()
+	{
+		return trackballView;
+	}
+
+	QString ViewerWidget::getHelpString()
+	{
+		return controlHelp;
+	}
+
+	bool ViewerWidget::isMovableByPicking(PhysicalObject* object)
+	{
+		std::map<PhysicalObject*, ExtendedAttributes>::iterator it = objectExtendedAttributesList.find(object);
+		if(it != objectExtendedAttributesList.end()) return it->second.movableByPicking;
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.